osc-staging.py

```diff
@@ -172,4 +174,9 @@ def do_staging(self, subcmd, opts, *args):
         elif cmd == 'list':
             ListCommand(api).perform()
         elif cmd == 'adi':
-            AdiCommand(api).perform(args[1:], opts.move)
+            if opts.move:
+                AdiCommand(api).perform(args[2:], opts.move)
+            elif opts.by_develproject:
+                AdiCommand(api).perform(args[2:], by_dp=opts.by_develproject)
+            else:
```

Comment:
I wonder why you move from args[1:] to args[2:] - and I think you can just have move=opts.move, by_dp=opts.by_develproject in one line without ifs